### PR TITLE
Building on both jdk8 and jdk11

### PR DESCRIPTION
**Pull request by Vlatombe**
This enables dual build on jdk8/jdk11.
Remove deprecated `useAci` option.

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by Vlatombe** [comment]
@jglick could you merge this to enable jdk11 build?

File: Jenkinsfile [hunk 1]
```diff
@@ -1 +1,5 @@
-buildPlugin(useAci: true)
+buildPlugin(configurations: [
+        [ platform: "linux", jdk: "8", jenkins: null ],
+        [ platform: "linux", jdk: "11", jenkins: null ],
+        [ platform: "windows", jdk: "8", jenkins: null, javaLevel: "8" ],
```
Comment (jglick) [inline Jenkinsfile:4]:
```suggestion
        [ platform: "linux", jdk: "8" ],
        [ platform: "linux", jdk: "11" ],
        [ platform: "windows", jdk: "8" ],
```

`javaLevel` is deprecated, and `jenkins: null` is a no-op.

File: Jenkinsfile [hunk 2]
```diff
@@ -1 +1,5 @@
-buildPlugin(useAci: true)
+buildPlugin(configurations: [
```
Comment (jglick) [inline Jenkinsfile:1]:
```suggestion
buildPlugin(useContainerAgent: true, configurations: [
```

**Review by jglick — COMMENTED** [review]
Can merge it if you promise to watch `master` builds and correct regressions, but it is probably best for me to push an origin PR based on this to test in advance.

**Comment by Vlatombe** [comment]
I ran jdk11 tests locally and it looked fine.

**Review by jglick — APPROVED** [review]
Looks right…

**Pull request merged**